nasa/cFE
# Fix #379: Remove test results from FSW path

**Describe the contribution**
Fixes #379 

**Testing performed**
Steps taken to test the contribution:
1. Checked out master bundle with this change, nominal setup (from README)
1. make ENABLE_UNIT_TESTS=true SIMULATION=native prep
1. make
1. make install
1. make test (timer test failed as expected, rest passed)
1. make lcov
no issues (and none expected since this wasn't a code change)

**Expected behavior changes**
 - No impact to behavior

**System(s) tested on:**
 - Hardware: cFS Dev Server
 - OS: Ubuntu 16.04
 - Versions: master bundle with this change

**Additional context**
None

**Contributor Info**
Jacob Hageman - NASA/GSFC
